impls/msft-green-ai.yaml

```diff
@@ -0,0 +1,44 @@
+name: mft-green-ai
```

Comment:
This one is impossible to model I think TBH, not in a way that exactly mirrors the case study. Too much data is missing from the case study. But we can play around and have a think about ways to make this interesting.

Reply:
yes, I just included a skeleton for completeness - agree it's not very useful yet.